### PR TITLE
Make aria-labels for links less verbose

### LINKED ISSUE #3137: aria-labels in footer are too verbose
**Expected behavior**
When the twitter, facebook, govdocs, and friends logos are given focus, a screen reader will read a concise description of the link, e.g. "PUL Twitter", "PUL Facebook", "Government Documents", "Friends of PUL".

These aria-labels are essentially acting as alt text, which [should be succinct](https://webaim.org/techniques/alttext/).

**Actual behavior**
The descriptions are not concise: "Opens Princeton University Library Facebook page", "Opens Princeton University Library Twitter page", "Opens Princeton University Library Government Documents Depository page", and "Opens Princeton University Library  Friends of the Library page".  Mentioning that it opens the particular page is redundant, since the screen reader will already tell users that it is a link.

**Steps to replicate**

1. Turn on voiceover
2. Press Ctl + Option + U to open the rotor
3. Use the Right/Left arrows to get the list of links
4. Use the Up/Down arrows to locate these links.  Note that the link text is quite verbose.
5. Press enter when one of the links is selected to set focus to it.  Note that it reads all the link text, plus mentioning that it is a link.

**Impact of this bug**
Screenreader experience is verbose and redundant.

### PR DESCRIPTION
**Pull request by maxkadel**
Closes #3137

### CODE REVIEW

File: app/views/shared/_footer.html.erb [hunk 1]
```diff
@@ -103,23 +103,23 @@
             <div class="footer--icons">
                 <a class="facebook-logo" href=
                 "https://www.facebook.com/PULibrary" aria-label=
-                "Opens Princeton University Library Facebook page">
+                "PUL Facebook">
```
Comment (christinach) [inline app/views/shared/_footer.html.erb:106]: The screen reader reads PUL which can be confusing for a screen reader user. Can you please rename PUL to Princeton University Library ?

File: app/views/shared/_footer.html.erb [hunk 2]
```diff
@@ -103,23 +103,23 @@
...
                 <%= image_tag('facebook.png', alt: "Facebook logo") %></a>
                 <a class="twitter-logo" href=
                 "https://twitter.com/PULibrary" aria-label=
-                "Opens Princeton University Library Twitter page">
+                "PUL Twitter">
```
Comment (christinach) [inline app/views/shared/_footer.html.erb:110]: The screen reader reads PUL which can be confusing for a screen reader user. Can you please rename PUL to Princeton University Library ?

**Review by christinach — CHANGES_REQUESTED** [review]
@maxkadel thank you for removing the redundant 'open' since it is a link, but I believe for the facebook and twitter icons we should keep the whole name of the library "Princeton University Library". 
When the screen reader reads PUL it is not clear that it refers to Princeton University Library.

File: app/views/shared/_footer.html.erb (hunk 1 above)
Comment (maxkadel) [reply app/views/shared/_footer.html.erb]: Done.

File: app/views/shared/_footer.html.erb (hunk 2 above)
Comment (maxkadel) [reply app/views/shared/_footer.html.erb]: Done.

**Review by christinach — APPROVED** [review]

**Pull request merged**